src/Discovery/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useEffect } from 'react';
+import { connect } from 'react-redux';
+
+import Discovery from './Discovery';
+import { DiscoveryConfig } from './DiscoveryConfig';
+import { userHasMethodForServiceOnResource } from '../authMappingUtils';
+import { hostname, discoveryConfig, useArboristUI } from '../localconf';
+
+const loadStudiesFromMDS = async (): Promise<any[]> => {
+  // Why `_guid_type='discovery_metadata'? We need to distinguish the discovery page studies in MDS
+  // from other data in MDS. So all MDS records with `_guid_type='discovery_metadata'` should be
+  // the full list of studies for this commons.
+  const GUID_TYPE = 'discovery_metadata';
+  const LIMIT = 10000; // required or else mds defaults to returning 10 records
```

Comment:
what if we have more than 10000 records in MDS? Does MDS support pagination queries?

Reply:
Thank you for calling this out -- MDS does support pagination, AND the max limit is actually 2000 records. I'll change the logic to this:
1. Get LIMIT=1000 records from mds. If request fails throw an error and show an empty page until user refreshes (no retry logic for right now)
2. While the number of records returned is 1000, query again with offset=1000 LIMIT=1000. If request fails throw an error and show an empty page until user refreshes (again no retry logic)